saleor/tests/e2e/orders/taxes/test_order_calculate_simple_tax_based_on_shipping_address_tax_class.py

```diff
@@ -117,26 +32,47 @@ def test_order_calculate_simple_tax_based_on_shipping_address_tax_class_CORE_200
         permission_manage_product_types_and_attributes,
         permission_manage_taxes,
         permission_manage_orders,
+        permission_manage_settings,
     ]
     assign_permissions(e2e_staff_api_client, permissions)
 
-    (
-        warehouse_id,
-        channel_id,
-        channel_slug,
-        shipping_method_id,
-        shipping_price,
-    ) = prepare_shop_with_few_shipping_zone_countries(e2e_staff_api_client, "6.66")
-
-    shipping_country_tax_rate = prepare_tax_configuration(
+    shop_data = prepare_shop(
         e2e_staff_api_client,
-        channel_slug,
+        shipping_zones_structure=[
+            {"countries": ["CZ", "DE", "US"], "num_shipping_methods": 1}
+        ],
         shipping_country_code="DE",
         shipping_country_tax_rate=19,
         billing_country_code="CZ",
         billing_country_tax_rate=21,
+        prices_entered_with_tax=False,
+        shipping_price=6.66,
+    )
+    channel_id = shop_data["channel_id"]
+    warehouse_id = shop_data["warehouse_id"]
+    shipping_price = shop_data["shipping_price"]
+    shipping_method_id = shop_data["shipping_method_id"]
+    shipping_country_tax_rate = shop_data["shipping_country_tax_rate"]
+    shipping_country_code = shop_data["shipping_country_code"]
+    shipping_tax_class_id = shop_data["shipping_tax_class_id"]
+    billing_country_code = shop_data["billing_country_code"]
+    billing_country_tax_rate = shop_data["billing_country_tax_rate"]
+    update_country_tax_rates(
+        e2e_staff_api_client,
+        shipping_country_code,
+        [{"rate": shipping_country_tax_rate}],
     )
 
+    update_shipping_price(
+        e2e_staff_api_client,
+        shipping_method_id,
+        {"taxClass": shipping_tax_class_id},
+    )
```

Comment:
The same as for core_2001
No need to update the shipping tax class. In this case tax rate for the shipping should be taken from the country rate
product tax class = null
product type tax class = null
shipping tax class = null
country rate for shipping country = 19
country rate for billing country = 21

product tax= variant price * country rate for shipping country
shipping tax = shipping price * country rate for shipping country